lib/irb/cmd/show_cmds.rb

```diff
@@ -16,6 +16,12 @@ def execute(*args)
         commands_info = IRB::ExtendCommandBundle.all_commands_info
         commands_grouped_by_categories = commands_info.group_by { |cmd| cmd[:category] }
 
+        user_aliases = irb_context.instance_variable_get(:@user_aliases)
```

Comment:
Since it's purely internal and only for this command at the moment, I don't plan to expose an API for it.

And I don't directly read `IRB.conf[:COMMAND_ALIASES]` here because if the user updates it in the session, help message and the actual behaviour will be out of sync.